src/ui/import/import_detail_view.rs

```diff
@@ -81,8 +81,10 @@ pub fn import_detail_view(id: usize, db: Db, que: Que) -> impl View {
 				let field_title_history = field_title.clone();
 				let dyn_field_kind = db.get_field_kind(&id, &field);
 				let is_secret = match dyn_field_kind {
```

Comment:
what is the determination between using `matches!` and a `match` ?

Reply:
Found the original clippy message in my buffer
<img width="1006" alt="image" src="https://github.com/dominikwilkowski/vault/assets/1266923/132d5aef-c331-4d59-ace0-663c80862798">